src/ParameterStack.ts

```diff
@@ -59,5 +60,46 @@ export class ParameterStack extends Stack {
     });
 
   }
+  private rxMissionZgwParameters() {
+
+    new StringParameter(this, 'ssm-rxmission-zgw-1', {
```

Comment:
Het format voor ZGW-endpoints staat vast, dus in principe zou 1 param per API/registratiecomponent genoeg moeten zijn, catalogi en zaken-baseurls zouden genoeg moeten zijn, de rest heeft vast format:
- https://catalogi.{{baseUrl}}/api/v1/catalogussen moet altijd op v1/catalogussen beschikbaar zijn
- https://catalogi.{{baseUrl}}/api/v1/zaaktypen moet altijd op /v1/zaaktypen beschikbaar zijn